src/components/ui/TaskCard.tsx

```diff
@@ -26,12 +39,20 @@ function TaskCard(props) {
     try {
       const requestBody = JSON.stringify(task);
       const response = await api.put(
-        `/api/v1/teams/${teamId}/tasks/${taskId}`,
-        requestBody
+        `/api/v1/teams/${teamId}/tasks/${task.taskId}`,
+        requestBody,
+        {
+          headers: {
+            Authorization: `${token}`,
+          },
+        }
       );
     } catch (error) {
-      console.error(`Failed to Update Task error ${handleError(error)}`);
+      console.error("Error moving Task:", handleError(error));
```

Comment:
Soft error handling: do we return to the user that something is happening or rather not happening?